### PR TITLE
Condense blog markdown information to one page

### PR DESCRIPTION
**Pull request by content-hacker**
add TOC, add editing instructions

### CODE REVIEW

**Comment by content-hacker** [comment]
@aileenrose I think I may have done something wrong here.. It looks like the PR wants to merge into my other PR instead of master. Can you help me fix this? 

Also, I want to "retire" the editing blog posts page since I added this information here. Can I delete that page from the repo, or what's the best way to do this?

File: handbook/marketing/creating_blog_posts.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+# Creating & editing blog posts with markdown and GitHub 
 
 We want and encourage everyone at Sourcegraph to contribute to the blog and publishing a blog post is as simple as merging an approved pull request with your blog content.
 
+## Table of contents 
+
+1. [Process overview](#process-overview)
+1. [How to create a new blog post in GitHub](#adding-a-blog-post)
+1. [How to add images to your blog post](#adding-images)
+1. [How to add a YouTube video to your blog post](youtube-video-embed-code)
```
Comment (aileenrose) [inline handbook/marketing/creating_blog_posts.md:10]:
```suggestion
1. [How to add a YouTube video to your blog post](#youtube-video-embed-code)
```
The missing hashtag is causing the error in the build tests.

Comment (aileenrose) [reply handbook/marketing/creating_blog_posts.md:10]: hello! @content-hacker see my suggested commit to resolve the error.

**Comment by aileenrose** [comment]
@content-hacker is this still in draft? I can review if ready. :)

**Comment by content-hacker** [comment]
@aileenrose sorry, I got completely sidetracked from handbook stuff. This is ready for review. I think I need to merge this into my other PR because of how I structured this branch? Does that sounds right to you?

**Review by aileenrose — APPROVED** [review]

**Pull request merged**